modules/labs-react/layout/stories/examples/Stack/StackItems.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import {Stack} from '@workday/canvas-kit-labs-react/layout';
 import {Box} from '@workday/canvas-kit-labs-react/common';
-import {Hyperlink} from '@workday/canvas-kit-react/button';
+import {Hyperlink, SecondaryButton} from '@workday/canvas-kit-react/button';
```

Comment:
Bug in the codemod. Need to undo this one

```suggestion
import {Hyperlink} from '@workday/canvas-kit-react/button';
```